src/precomputed-requestor.ts

```diff
@@ -19,6 +27,14 @@ export default class PrecomputedFlagRequestor {
       subject_attributes: this.subjectAttributes,
     });
 
+    if (this.onPrecomputedResponse && precomputedResponse) {
+      this.onPrecomputedResponse({
+        salt: precomputedResponse.salt,
+        subjectKey: this.subjectKey,
+        subjectAttributes: this.subjectAttributes,
+      });
+    }
+
     if (!precomputedResponse?.flags) {
       return;
     }
```

Comment:
minor: flags are non-optional, so this check is no-op

Reply:
🧹